tests/test_tracking.py

```diff
@@ -473,3 +484,41 @@ def test_log(self):
                     "some_string": "",
                 }
                 self.assertDictEqual(data, truth)
+
+
+@require_dvclive
+@mock.patch("dvclive.live.get_dvc_repo", return_value=None)
```

Comment:
@dberenbaum FYI we're seeing failures on the transformers side in the example scripts because of this. Is there a better way to ignore the git repo than patching for cases where we're running a full training script? (Like via an env variable or something). Otherwise I'll have to disable dvc for all of the example script tests in `transformers` which wouldn't be ideal. https://github.com/huggingface/accelerate/actions/runs/7023901505/job/19111477258

Reply:
I need to take a closer look when I have a minute, but you can use `DVCLIVE_TEST = true` to disable the git-related functionality. For example:

https://github.com/huggingface/accelerate/blob/31c723febe6d37f2c318c90bb6bebec64c8df035/tests/test_examples.py#L208

Reply:
Ah I missed that one, fantastic! That should do the job just fine I think. I’ll get to that tommorow and cc you. Thanks!

Reply:
Is https://github.com/huggingface/accelerate/pull/2196 related?

Reply:
It’s not, we saw dependency errors in an isolated env related to the release

Reply:
Thanks. Added a comment there since it should be fixed now. Let me know if it would be helpful to submit a PR for either or if it's quicker to take care of it yourself.